codalab/lib/worksheet_util.py

```diff
@@ -80,6 +83,126 @@ def worksheet_line(description, uuid):
     return '[%s]{{%s}}' % (description, uuid)
 
 
+
+############################################################
+
+
+class WorksheetResolver(object):
```

Comment:
@percyliang In the server case, we want to just call the model methods directly. In the client case, we have to make calls to the REST API.

I'll take a shot at cleaning this up!